draft-regext-rdap-extensions.md

```diff
@@ -276,7 +285,11 @@ be the extension identifier in cases where the extension defines only one object
 Because the `objectClassName` is a string and [@!RFC9083] sets the precedent of using
 spaces in object class names (i.e. "ip network"), extensions may follow the same
 convention. It is RECOMMENDED that object class names be lower cased, ASCII characters
-that use the space character as a word separator.
+that use either the space character or the "\_" (underscore) character as a word separator.
+Usage of the space character as a word separator follows existing convention, but usage
+of the "\_" (underscore) character may be preferred when the name is also used in URI paths.
+When object class names are also used in URIs, extensions MUST specify that the name
+is to be URL-encoded as defined in [@!RFC3986].
```

Comment:
I don't think this is aligned with the earlier 'usage in queries'
section, which notes that path segments have to be equal to the
identifier or prepended with '{identifier}_'.  I think it would be
simpler here to disallow the use of whitespace in 'objectClassName'.

Reply:
> I think it would be simpler here to disallow the use of whitespace in 'objectClassName'.

But, would that not making things backwards-incompatible vis-a-vis the "ip network" object class name?

Reply:
After going back and forth with this, I agree with Tom that it is more straight forward with one set of rules. We can call out "ip network" as an anomaly as it never appears in the any query paths.

Reply:
Okie dokie.

Reply:
I think I have addressed this.